package.json

```diff
@@ -66,6 +66,7 @@
     "jest": "29.7.0",
     "js-beautify": "1.15.1",
     "markdown-spellcheck": "1.3.1",
+    "mime": "3.0.0",
```

Comment:
What are you using this new dependency for? I tried running the PR locally without this dependency and things seem to work OK.

Reply:
Nvm, looks like this is used in an unchanged line of the converter script, and didn't exist as a dev dependency before

Reply:
Yes, not sure why it was not added before, as it's directly required in the script. Maybe it used to be a dependency of something else, so used to be available even if not explicitly listed. Must have disappeared in the meantime.

Bugs me a bit to add it just for this script that is used so rarely, but it's just a dev dep, so doesn't hurt anyone really (except us).